src/benchmark_api_internal.h

```diff
@@ -10,6 +12,7 @@
 
 #include "benchmark/benchmark.h"
 #include "commandlineflags.h"
+#include "string_util.h"
```

Comment:
are these headers needed in this file?  i don't think there's a dependency being added to the interface.